tests/bgp/test_bgp_azng_migration.py

```diff
@@ -158,8 +158,36 @@ def test_bgp_azng_migration(duthosts, enum_upstream_dut_hostname):
                 assert routes_json['totalPrefixCounter'] == original_ipv6_route_adv_count
                 assert routes_json['filteredPrefixCounter'] == original_ipv6_route_adv_filter_count
             else:
-                assert routes_json['totalPrefixCounter'] == original_ipv4_route_adv_count
-                assert routes_json['filteredPrefixCounter'] == original_ipv4_route_adv_filter_count
+                assert routes_json['totalPrefixCounter'] == original_ipv4_route_adv_count, (
+                    "Mismatch in total advertised IPv4 route count after AZNG migration. "
+                    "Expected: {}, Actual: {}. "
+                    "- original_ipv4_route_adv_count: {}\n"
+                    "- routes_json['totalPrefixCounter']: {}\n"
+                    "- Command: {}\n"
+                    "- routes_json: {}"
+                ).format(
+                  original_ipv4_route_adv_count,
+                  routes_json['totalPrefixCounter'],
+                  original_ipv4_route_adv_count,
+                  routes_json['totalPrefixCounter'],
+                  bgp_nbr_adv_cmd,
+                  routes_json
+                )
+                assert routes_json['filteredPrefixCounter'] == original_ipv4_route_adv_filter_count, (
+                   "Mismatch in filtered advertised IPv4 route count after AZNG migration. "
+                   "Expected: {}, Actual: {}. "
+                   "- original_ipv4_route_adv_filter_count: {}\n"
+                   "- routes_json['filteredPrefixCounter']: {}\n"
+                   "- Command: {}\n"
+                   "- routes_json: {}"
+                ).format(
+                  original_ipv4_route_adv_filter_count,
+                  routes_json['filteredPrefixCounter'],
+                  original_ipv4_route_adv_filter_count,
+                  routes_json['filteredPrefixCounter'],
```

Comment:
original_ipv4_route_adv_filter_count, routes_json['filteredPrefixCounter'] were printed out, why are you keeping printing them again? If those variable are very long, the summary can't be readable.